readme.md

```diff
@@ -351,6 +351,18 @@
 | ------: | -------: | -----: |
 |   2 - 4 |        8 | 30-60m |
 
+### [Mahjong Solitaire](https://boardgamegeek.com/boardgame/67948/mahjong-solitaire)
+
+>Mahjong Solitaire is a popular single-player tile-matching game, distinct from traditional Mahjong. The objective is to clear the board by matching identical, "open" tiles. Tiles must have no obstructions on their sides or on top. Special tiles, like flowers or seasons, can boost your score. Some versions have a timer, challenging you to clear the board before it runs out. Strategy and planning are crucial for success. It's a relaxing and mentally stimulating game enjoyed by millions.
```

Comment:
The linter action is failing due to the issue on this line. After the `>` it needs to have a space to be properly formatted.